code/modules/mob/living/living.dm

```diff
@@ -797,22 +797,21 @@ default behaviour is:
 
 /mob/living/handle_drowning()
 	var/turf/T = get_turf(src)
-	if(!can_drown() || !loc.is_flooded(lying))
+	if(!T || !can_drown() || !T.is_flooded(lying))
```

Comment:
~~you could be on the bottom of the ocean and not drown if you're in a locker with this change.~~ `loc.is_flooded(lying)` was correct I think

actually i think this has the opposite issue. atoms can't be not-flooded if their turf is flooded, which might cause issues for airtight lockers, exosuits, etc.